src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -467,6 +472,61 @@ bool CascadiaSettings::_isPowerShellCoreInstalledInPath(const std::wstring_view
     return false;
 }
 
+// Function Description:
+// - Adds all of the WSL profiles to the provided container.
+// Arguments:
+// - A ref to the profiles container where the WSL profiles are to be added
+// Return Value:
+// - <none>
+void CascadiaSettings::_CreateWslProfiles(std::vector<TerminalApp::Profile>& profileStorage)
+{
+    try
+    {
+        DWORD bytesAvailable = 0;

```

Comment:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es21-dont-introduce-a-variable-or-constant-before-you-need-to-use-it

several variables can be moved closer to its first use

Reply:
Ah!  Thanks for the update!